Computer-SpaceGames/Starship-Takeoff/cpp/src/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <random>
+#include "game.h"
+
+int main(int argc, char const *argv[])
+{
+    std::random_device rd;
+    std::mt19937_64 eng(rd());
+
+    std::uniform_int_distribution<unsigned int> gravity_distr{1, 20};
+    std::uniform_int_distribution<unsigned int> weight_distr{1, 40};
+
+    auto gravity =  gravity_distr(eng);
+    auto weight =  weight_distr(eng);
+
+    SpaceTakeoffGame game(gravity, weight);
+
+    std::cout << "STARSHIP TAKE-OFF" << std::endl;
+    std::cout << "GRAVITY=" << gravity << std::endl;
+    std::cout << "TYPE IN FORCE" << std::endl;
+    while (true) {
```

Comment:
I think `do { ... } while (!game.over());` would be better than `while(true)` with a `break`.